test/component-movable-test.js

```diff
@@ -22,6 +22,16 @@ describe('Moveable component', () => {
     viewInstance.remove();
   }
 
+  function mouseMoveTo(x, y) {
```

Comment:
So maybe somthing like `dragAndDropMock` or something like that, because it in fact mock dragging and dropping, not only moving mouse.